src/user_interface/configPopup.py

```diff
@@ -0,0 +1,56 @@
+from PyQt5 import QtGui, QtCore
+from PyQt5.QtWidgets import QDialog, QWidget, QVBoxLayout, QHBoxLayout, QPushButton, QStackedLayout
+from src.user_interface.iconsList import IconsList
+from src.user_interface.piano import Piano
+from PyQt5.QtCore import Qt, QRect
+
+
+class ConfigPopup(QDialog):
+
+    def __init__(self):
+        super().__init__()
+        stacked = QStackedLayout()
+        gray = QRect()
+        layout = QVBoxLayout()
+        self.setWindowFlags(QtCore.Qt.FramelessWindowHint)
+        self.setGeometry(100, 100, 500, 200)
+        layout.addLayout(IconsList())
+        layout.addLayout(Piano())
```

Comment:
Actually I lied, one more thing. This looks good on my desktop, but on my laptop's smaller screen it overlaps a few other things. Can you put a boarder on it so it looks a little cleaner?

Reply:
I put the default windowed boarder around it, in the future I want to make it so it grays out the main window, which will prevent this popup from blending in the background too much